docs/sections/user_guide/cli/mode_forecast.rst

```diff
@@ -55,4 +55,38 @@ The ``uw`` mode for configuring and running forecasts.
 Examples
 ^^^^^^^^
 
-TBD
+The examples use a configuration file named ``config.yaml``. Its contents are described in depth in Section :ref:`forecast_yaml`.
+
+* Run an FV3 forecast on an interactive node
+
+  .. code-block:: sh
+
+    $ uw forecast run -c config.yaml --cycle 2024-01-09T12 --model FV3
+
+  The forecast will run on the node where you have invoked this command. Optionally, capture the output in a log file using shell redirection.
+
+* Run an FV3 forecast using a batch system 
+
+  .. code-block:: sh
+
+    $ uw forecast run -c config.yaml --cycle 2024-01-09T12 --model FV3 --batch-script submit_fv3.sh
+
+  This command writes a file named ``submit_fv3.sh`` and submits it to the batch system.
+
+* With the ``--dry-run`` flag specified, nothing is written to ``stdout``, but a report of what would have been written is logged to ``stderr``:
```

Comment:
I feel like, normally, the purpose of `--dry-run` is to stop meaningful things from happening, vs to stop things from being printed to the console. Is that really all `--dry-run` does for us in this mode at the moment? If so, I guess this is what we need to say. It does seem like an odd use of a dry-run flag, though. For example, as a user I think I'd expect that, say, `--dry-run` will show all the directories, files, symlinks, etc., that would have been created, but not actually create them and/or create a batch script but not submit it. I mainly wanted to make sure that this statement is at least _accurate_ for now, even if the actual behavior isn't what we'd hope.

I think the DAG work will make dry-run behavior more realistic. For example, in the first graph example shown in https://github.com/maddenp/iotaa?tab=readme-ov-file#graphing, the disk is completely untouched. Hopefully we can get similar behavior here, assuming that's what we'd want.

Reply:
That works for me. I've updated the sentence here to be more accurate (hopefully).